### DISCUSSION TITLE
Notification clean

### DISCUSSION THREAD

**Discussion by Decodetalkers** [XAMPPRocky/octocrab#303] [Q&A]
I have read the example of notifications, but I don't know how to clean any of a notification with octocrab

**Comment by envp** [comment]
If you meant to interact with notifications maybe this would help? https://docs.github.com/en/rest/activity/notifications?apiVersion=2022-11-28

I haven't looked at the code so I'm not sure if _all_ of these are implemented, but you could give it a shot. The internals are quite easy to work with :)

**Reply by Decodetalkers** [reply]
> If you meant to interact with notifications maybe this would help? https://docs.github.com/en/rest/activity/notifications?apiVersion=2022-11-28
> 
> I haven't looked at the code so I'm not sure if _all_ of these are implemented, but you could give it a shot. The internals are quite easy to work with :)

Thanks for your reply.. and I have already know how to do

just

```rust
 octocrab
          .activity()
           .notifications()
           .mark_all_as_read(None)
           .await?;
```
Thanks
